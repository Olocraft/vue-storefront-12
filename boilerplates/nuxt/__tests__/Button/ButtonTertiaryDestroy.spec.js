import { shallowMount } from '@vue/test-utils';
import ButtonDestroyTertiary from '../../components/Button/Tertiary/Destroy/ButtonTertiaryDestroy.vue';

let wrapper;

describe('ButtonDestroyTertiary.vue', () => {
  beforeEach(() => {
    wrapper = shallowMount(ButtonDestroyTertiary);
  });
  it('render component correctly', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
