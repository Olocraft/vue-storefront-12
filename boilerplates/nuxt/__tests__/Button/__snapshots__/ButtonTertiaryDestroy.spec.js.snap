// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonDestroyTertiary.vue render component correctly 1`] = `<button class="inline-flex items-center justify-center cursor-pointer bg-transparent hover:bg-rose-100 active:bg-rose-200 disabled:bg-transparent text-rose-600 hover:text-rose-700 active:text-rose-800 disabled:text-gray-500 disabled:opacity-50 font-body text-base font-medium px-[16px] py-2 rounded-md outline-violet"></button>`;
